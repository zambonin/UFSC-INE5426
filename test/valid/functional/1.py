exec(open('../../../src/scope_manager.py', 'r').read())



a = 2
b = 4
def λ(x, y):
    return (x + y)

c = λ(a, b)
