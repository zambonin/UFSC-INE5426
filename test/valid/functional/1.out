int var: a, b, c
= a 2
= b 4
int fun: lambda (params: int x, int y)
  ret + x y
= c lambda[2 params] a b
